lib/explorer/series.ex

```diff
@@ -1864,6 +1864,57 @@ defmodule Explorer.Series do
   def quantile(%Series{dtype: dtype}, _),
     do: dtype_error("quantile/2", dtype, [:integer, :float, :date, :time, :datetime])
 
+  @doc """
+  Compute the sample skewness of a data set.
+
+  For normally distributed data, the skewness should be about zero.
+  For unimodal continuous distributions, a skewness value greater
+  than zero means that there is more weight in the right tail of the
+  distribution. The function skewtest can be used to determine if
```

Comment:
What is the `skewtest` function?

Reply:
Removed the reference to `skewtest` as it seems to just be available on [scipy](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.skewtest.html#scipy.stats.skewtest).